package.json

```diff
@@ -19,7 +19,7 @@
     "node": ">=0.8"
   },
   "dependencies": {
-    "backbone": "1.1.2",
+    "backbone": "1.2.3",
```

Comment:
Thoughts on not locking it in? 1.2.4 is right around the corner: jashkenas/backbone#3921